js/cryptopia.js

```diff
@@ -557,7 +563,7 @@ module.exports = class cryptopia extends Exchange {
                 }
             }
             let order = this.orders[id];
-            if (order['symbol'] === symbol)
+            if (symbol === undefined || order['symbol'] === symbol)
```

Comment:
`symbol === undefined` is not transpileable, but `typeof symbol === 'undefined'` is, can you change this line like that plz? This would fix the issue )

Reply:
Sure, I'll change it.